update sso docs (okta) with extra detail around account provisioning (#2005)
buildkite/docs: pages/integrations/sso/okta.md

### Provisioning existing users

> Existing Okta users aren't automatically provisioned in Buildkite; you'll need to sync your users in order to deprovision them. Users will have their Buildkite accounts created via [JIT provisioning](/docs/integrations/sso#frequently-asked-questions-do-you-support-jit-provisioning).

Reviewer: This reads better to me, but I'm not sure if it's correct. Can you check this conveys the same meaning, or let me know what's not right?

```suggestion
Buildkite creates accounts for existing Okta users with just-in-time user provisioning (JIT provisioning). To deprovision users, you need to sync them.
```

Author: @mbelton-buildkite These are much better words! Thanks for checking :)
